engine/Parser/Checks/CheckForIllegalCharacters.py

```diff
@@ -23,9 +23,9 @@ def check_for_illegal_characters(statement):
 		if validate_input(char):
 			continue
 		else:
-			logger.exception("String contains illegal characters, please check and try again: char=%s", char)
+			logger.exception("FAIL: String contains illegal characters, please check and try again: char=%s", char)
 			raise IllegalCharactersException("String contains illegal characters, please check and try again")
```

Comment:
Not sure if an exception is supposed to ask you to try again.

Reply:
This is something we'd see in the UI. Or CLI.

Reply:
Changed.